main.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"net/http"
+
+	handler "github.com/livepeer/leaderboard-serverless/api"
+	"github.com/livepeer/leaderboard-serverless/common"
+)
+
+// this func is for running in local mode.  Vercel does not use this as an entrypoint
+// so any logic here should only reflect what is needed for local development
+func main() {
+
+	http.HandleFunc("/api/raw_stats", handler.RawStatsHandler)
+	http.HandleFunc("/api/aggregated_stats", handler.AggregatedStatsHandler)
```

Comment:
Coudl you add a comment to each endpoint where it is used? I know, e.g., that `/aggregated_stats` is used in Explorer and Livepeer Studio Broadcaster. But I'm not sure about other ednpoints, like `/api/top_ai_score` or `/api/pipelines` pr `/api/regions`.

Reply:
I am not sure where all the endpoints are used.  Any new ones we added are for Explorer.  In addition, I am afraid  comments eventually drift over time and create a false sense of security for someone refactoring it.